src/core/thread/mle.cpp

```diff
@@ -1484,6 +1485,11 @@ void Mle::HandleStateChanged(otChangedFlags aFlags)
 #endif
     }
 
+    if (mChildUpdateRequestState == kChildUpdateRequestPending)
+    {
+        mChildUpdateRequestTimer.Start(kChildUpdateRequestPendingDelay);
```

Comment:
How about if we limit the restart of the timer only to `OT_CHANGED_<>` events  that require a child update to be sent?
My concern is that unrelated notification events may restart the timer and delay the tx of child update.

Thoughts?

Reply:
make sense.